src/plugins/template/tests/functional/shared_tests_instances/execution_graph_tests/duplicate_inputs_outputs_names.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "execution_graph_tests/duplicate_inputs_outputs_names.hpp"
+
+#include "common_test_utils/test_constants.hpp"
+
+using namespace ExecutionGraphTests;
+
+namespace {
+
+INSTANTIATE_TEST_SUITE_P(smoke_duplicateInputsOutputsNames,
+                         ExecGraphDuplicateInputsOutputsNames,
+                         ::testing::Values(ov::test::utils::DEVICE_TEMPLATE),
+                         ExecGraphDuplicateInputsOutputsNames::getTestCaseName);
+
+}  // namespace
```

Comment:
I require some help with this one, the test runs fine locally but on CI I see it's failling with the following error:
'''
2024-05-29T13:59:22.0457469Z [ RUN      ] smoke_duplicateInputsOutputsNames/ExecGraphDuplicateInputsOutputsNames.CheckOutputsMatch/Dev=TEMPLATE
2024-05-29T13:59:22.0473250Z unknown file: error: C++ exception with description "Exception from openvino\openvino\src\inference\src\cpp\core.cpp:107:
2024-05-29T13:59:22.0475439Z Exception from openvino\openvino\src\inference\src\dev\core_impl.cpp:566:
2024-05-29T13:59:22.0476723Z Device with "TEMPLATE" name is not registered in the OpenVINO Runtime
2024-05-29T13:59:22.0477518Z 
2024-05-29T13:59:22.0477717Z " thrown in the test body.
2024-05-29T13:59:22.0479664Z [  FAILED  ] smoke_duplicateInputsOutputsNames/ExecGraphDuplicateInputsOutputsNames.CheckOutputsMatch/Dev=TEMPLATE, where GetParam() = "TEMPLATE" (2 ms)
'''